<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncomeMan</class>
 <widget class="QDialog" name="IncomeMan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1102</width>
        <height>3022</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>1080</width>
           <height>3000</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="horizontalFrame">
          <property name="geometry">
           <rect>
            <x>-40</x>
            <y>9</y>
            <width>551</width>
            <height>501</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout"/>
         </widget>
         <widget class="QFrame" name="horizontalFrame_2">
          <property name="geometry">
           <rect>
            <x>209</x>
            <y>1690</y>
            <width>661</width>
            <height>411</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2"/>
         </widget>
         <widget class="QFrame" name="horizontalFrame3">
          <property name="geometry">
           <rect>
            <x>520</x>
            <y>9</y>
            <width>541</width>
            <height>501</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4"/>
         </widget>
         <widget class="QFrame" name="horizontalFrame4">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>1210</y>
            <width>661</width>
            <height>661</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3"/>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>510</y>
            <width>481</width>
            <height>481</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background:rgba(34, 34, 34,78%);
border-radius:20px;
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="RoomManagement">
          <property name="geometry">
           <rect>
            <x>63</x>
            <y>628</y>
            <width>219</width>
            <height>25</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift SemiBold</family>
            <pointsize>15</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 600 15pt &quot;Bahnschrift SemiBold&quot;;</string>
          </property>
          <property name="text">
           <string>Room Management:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>627</y>
            <width>154</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="Entertainment">
          <property name="geometry">
           <rect>
            <x>92</x>
            <y>678</y>
            <width>171</width>
            <height>25</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift SemiBold</family>
            <pointsize>15</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 600 15pt &quot;Bahnschrift SemiBold&quot;;</string>
          </property>
          <property name="text">
           <string> Entertainment:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>678</y>
            <width>154</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="Food">
          <property name="geometry">
           <rect>
            <x>152</x>
            <y>728</y>
            <width>61</width>
            <height>25</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift SemiBold</family>
            <pointsize>15</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 600 15pt &quot;Bahnschrift SemiBold&quot;;</string>
          </property>
          <property name="text">
           <string>Food:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>729</y>
            <width>154</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="RoomBooking">
          <property name="geometry">
           <rect>
            <x>112</x>
            <y>778</y>
            <width>171</width>
            <height>25</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 600 15pt &quot;Bahnschrift SemiBold&quot;;
color:rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Room Booking:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>780</y>
            <width>154</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>940</y>
            <width>154</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>880</y>
            <width>381</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>940</y>
            <width>151</width>
            <height>25</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 600 15pt &quot;Bahnschrift SemiBold&quot;;
color:rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Total Earned:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>550</y>
            <width>161</width>
            <height>61</height>
           </rect>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::PreventContextMenu</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 600 25pt &quot;Bahnschrift SemiBold&quot;;
color:rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Income </string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>510</y>
            <width>491</width>
            <height>691</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background:rgba(34, 34, 34,78%);
border-radius:20px;
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_6">
          <property name="geometry">
           <rect>
            <x>830</x>
            <y>627</y>
            <width>154</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>610</x>
            <y>1030</y>
            <width>381</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>660</x>
            <y>1110</y>
            <width>151</width>
            <height>25</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 600 15pt &quot;Bahnschrift SemiBold&quot;;
color:rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Total Earned:</string>
          </property>
         </widget>
         <widget class="QLabel" name="RoomManagement_2">
          <property name="geometry">
           <rect>
            <x>603</x>
            <y>628</y>
            <width>219</width>
            <height>25</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift SemiBold</family>
            <pointsize>15</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 600 15pt &quot;Bahnschrift SemiBold&quot;;</string>
          </property>
          <property name="text">
           <string>Room Management:</string>
          </property>
         </widget>
         <widget class="QLabel" name="Entertainment_2">
          <property name="geometry">
           <rect>
            <x>620</x>
            <y>678</y>
            <width>171</width>
            <height>25</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift SemiBold</family>
            <pointsize>15</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 600 15pt &quot;Bahnschrift SemiBold&quot;;</string>
          </property>
          <property name="text">
           <string> Entertainment:</string>
          </property>
         </widget>
         <widget class="QLabel" name="Food_2">
          <property name="geometry">
           <rect>
            <x>600</x>
            <y>728</y>
            <width>221</width>
            <height>25</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift SemiBold</family>
            <pointsize>15</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 600 15pt &quot;Bahnschrift SemiBold&quot;;</string>
          </property>
          <property name="text">
           <string>Food And Bevrages</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_7">
          <property name="geometry">
           <rect>
            <x>830</x>
            <y>1110</y>
            <width>154</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="RoomBooking_2">
          <property name="geometry">
           <rect>
            <x>610</x>
            <y>778</y>
            <width>231</width>
            <height>25</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 600 15pt &quot;Bahnschrift SemiBold&quot;;
color:rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Staff Management</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_8">
          <property name="geometry">
           <rect>
            <x>830</x>
            <y>729</y>
            <width>154</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_9">
          <property name="geometry">
           <rect>
            <x>830</x>
            <y>678</y>
            <width>154</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_10">
          <property name="geometry">
           <rect>
            <x>830</x>
            <y>780</y>
            <width>154</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>680</x>
            <y>550</y>
            <width>241</width>
            <height>61</height>
           </rect>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::PreventContextMenu</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 600 25pt &quot;Bahnschrift SemiBold&quot;;
color:rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Expenditure</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="RoomBooking_3">
          <property name="geometry">
           <rect>
            <x>610</x>
            <y>824</y>
            <width>211</width>
            <height>25</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 600 15pt &quot;Bahnschrift SemiBold&quot;;
color:rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Tax And Insurance</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_11">
          <property name="geometry">
           <rect>
            <x>830</x>
            <y>826</y>
            <width>154</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="RoomBooking_4">
          <property name="geometry">
           <rect>
            <x>670</x>
            <y>870</y>
            <width>171</width>
            <height>25</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 600 15pt &quot;Bahnschrift SemiBold&quot;;
color:rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Utilities</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_12">
          <property name="geometry">
           <rect>
            <x>830</x>
            <y>872</y>
            <width>154</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="RoomManagement_3">
          <property name="geometry">
           <rect>
            <x>591</x>
            <y>920</y>
            <width>231</width>
            <height>25</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift SemiBold</family>
            <pointsize>15</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 600 15pt &quot;Bahnschrift SemiBold&quot;;</string>
          </property>
          <property name="text">
           <string>Marketing And Sales</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_13">
          <property name="geometry">
           <rect>
            <x>830</x>
            <y>920</y>
            <width>154</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_14">
          <property name="geometry">
           <rect>
            <x>829</x>
            <y>970</y>
            <width>154</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="RoomManagement_4">
          <property name="geometry">
           <rect>
            <x>590</x>
            <y>970</y>
            <width>241</width>
            <height>25</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift SemiBold</family>
            <pointsize>15</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 600 15pt &quot;Bahnschrift SemiBold&quot;;</string>
          </property>
          <property name="text">
           <string>System Maintainance</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>1020</y>
            <width>481</width>
            <height>181</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background:rgba(34, 34, 34,78%);
border-radius:20px;
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="RoomBooking_5">
          <property name="geometry">
           <rect>
            <x>112</x>
            <y>828</y>
            <width>171</width>
            <height>25</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 600 15pt &quot;Bahnschrift SemiBold&quot;;
color:rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Other Sources</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_15">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>830</y>
            <width>154</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="RoomBooking_6">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>1050</y>
            <width>171</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift SemiBold</family>
            <pointsize>25</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 600 25pt &quot;Bahnschrift SemiBold&quot;;
color:rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Profit</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_16">
          <property name="geometry">
           <rect>
            <x>153</x>
            <y>1110</y>
            <width>251</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:15px;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>93</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
         <zorder>label</zorder>
         <zorder>horizontalFrame</zorder>
         <zorder>horizontalFrame_2</zorder>
         <zorder>horizontalFrame3</zorder>
         <zorder>horizontalFrame4</zorder>
         <zorder>RoomManagement</zorder>
         <zorder>lineEdit</zorder>
         <zorder>Entertainment</zorder>
         <zorder>lineEdit_2</zorder>
         <zorder>Food</zorder>
         <zorder>lineEdit_3</zorder>
         <zorder>RoomBooking</zorder>
         <zorder>lineEdit_4</zorder>
         <zorder>lineEdit_5</zorder>
         <zorder>pushButton</zorder>
         <zorder>label_2</zorder>
         <zorder>label_3</zorder>
         <zorder>label_4</zorder>
         <zorder>lineEdit_6</zorder>
         <zorder>pushButton_2</zorder>
         <zorder>label_5</zorder>
         <zorder>RoomManagement_2</zorder>
         <zorder>Entertainment_2</zorder>
         <zorder>Food_2</zorder>
         <zorder>lineEdit_7</zorder>
         <zorder>RoomBooking_2</zorder>
         <zorder>lineEdit_8</zorder>
         <zorder>lineEdit_9</zorder>
         <zorder>lineEdit_10</zorder>
         <zorder>label_7</zorder>
         <zorder>RoomBooking_3</zorder>
         <zorder>lineEdit_11</zorder>
         <zorder>RoomBooking_4</zorder>
         <zorder>lineEdit_12</zorder>
         <zorder>RoomManagement_3</zorder>
         <zorder>lineEdit_13</zorder>
         <zorder>lineEdit_14</zorder>
         <zorder>RoomManagement_4</zorder>
         <zorder>label_6</zorder>
         <zorder>RoomBooking_5</zorder>
         <zorder>lineEdit_15</zorder>
         <zorder>RoomBooking_6</zorder>
         <zorder>lineEdit_16</zorder>
         <zorder>pushButton_3</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
